telco-examples/edge-clusters/dhcp/eib/custom/scripts/fix-growfs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I'd say to add a number prefix just to align on what EIB expects.

Reply:
Also add a link to the BZ where this is being tracked

Reply:
ok, changed